Dockerfile

```diff
@@ -0,0 +1,25 @@
+#
+# Build
+#
+FROM maven:3.8.4-jdk-11-slim as buildtime
+WORKDIR /build
+COPY . .
+RUN mvn clean package -Dmaven.test.skip=true
+
+#
+# Package stage
+#
+FROM --platform=linux/amd64 adoptopenjdk/openjdk16:alpine as builder
+COPY --from=buildtime /build/target/*.jar application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+
```

Comment:
do we want to add APM agent too?

Reply:
Is it not already included in this [image](https://github.com/pagopa/docker-base-springboot-openjdk11/blob/eda9cbeb7a0466641e5f173952286958f7abf5b8/Dockerfile#L7C1-L7C1)?